src/filesystem_wrapper.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "disasm.h"
```

Comment:
This shouldn't be here

Reply:
@fbs @danobi Hi, removed with the latest commit